<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskMonitor</class>
 <widget class="QDialog" name="TaskMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTreeWidget" name="resultTree">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionMore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskMonitor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskMonitor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
